packages/sitebuilder/src/wizards/first-time-configuration/components/CustomSelectInput.tsx

```diff
@@ -85,9 +90,9 @@ export default function CustomSelectInput(props: CustomSelectInterface) {
 				} }
 			>
 				{ options.map((item) => (
-					<MenuItem key={ item.name } onClick={ () => handleClose(item.name) }>
-						<span style={ { fontWeight: 700 } }>{ item.name }</span>
-						&nbsp;{ item.description }
+					<MenuItem key={ item.label } onClick={ () => handleClose(item.value) }>
+						<span style={ { fontWeight: 700 } }>{ item.label }</span>
+						&nbsp;{ item.labelDescription }
```

Comment:
Could be better to add a condition here to check the existence of labelDescription value since it's optional according to the interface.